packages/core/content-manager/server/src/history/services/lifecycles.ts

```diff
@@ -131,13 +131,20 @@ const createLifecyclesService = ({ strapi }: { strapi: Core.Strapi }) => {
         // All schemas related to the content type
         const uid = context.contentType.uid;
         const schemas = getSchemas(uid);
+        const model = strapi.getModel(uid);
+
+        const isLocalizedContentType = serviceUtils.isLocalizedContentType(model);
 
         // Find all affected entries
         const localeEntries = await strapi.db.query(uid).findMany({
           where: {
             documentId,
-            locale: { $in: locales },
-            publishedAt: null,
+            ...(isLocalizedContentType
+              ? {
+                  locale: { $in: locales },
+                  publishedAt: null,
+                }
+              : {}),
```

Comment:
the published at filter should always apply, wether the content type is localized or not

Reply:
But entries without draft & publish are always published, so they have publishedAt with a date value (same as created I think)

Reply:
true, then that would require another check, wether the content type has D&P enabled